pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/store/IndexEntry.java

```diff
@@ -27,7 +28,7 @@
 class IndexEntry {
   private static Logger LOGGER = LoggerFactory.getLogger(IndexEntry.class);
 
-  IndexKey key;
+  final IndexKey key;
```

Comment:
Since we are touching this file, let's fix the format to prefix the member variables with underscore and remove `this.`

Reply:
ack

Reply:
renamed them here https://github.com/apache/pinot/pull/7316 to keep this PR short.